nemo/export/vllm/model_config.py

```diff
@@ -134,6 +135,9 @@ def __init__(
             disable_sliding_window=self.disable_sliding_window,
             sliding_window_len=self.get_hf_config_sliding_window(),
         )
+        self.is_attention_free = self._init_attention_free()
+        self.has_inner_state = self._init_has_inner_state()
```

Comment:
Needed for vllm >= 0.6.3 https://github.com/vllm-project/vllm/blob/v0.6.3/vllm/config.py#L199-L200

Reply:
Does this raise error with the vllm = 0.6.0?

Reply:
No, it's all OK for 0.6.0. It will be needed on bumping vLLM to latest version which I'm planning to do.